src/app/manage/ApplicationSettings.ts

```diff
@@ -81,6 +81,18 @@ export class ApplicationSettings extends Entity<number>  {
       this.validateSmsContent(this.reminderSmsText);
     }
   });
+  donorEmailText = new StringColumn({
```

Comment:
Terminology should be family and not donor - donor is specific to mlt, and the code in general is for families.

this column should be named "registerFamilyEmailText"